src/Chapter4.hs

```diff
@@ -629,7 +650,8 @@ Can you implement a monad version of AND, polymorphic over any monad?
 🕯 HINT: Use "(>>=)", "pure" and anonymous function
 -}
 andM :: (Monad m) => m Bool -> m Bool -> m Bool
-andM = error "andM: Not implemented!"
+andM fa fb = fa >>= (\a -> if a then fb else pure False)
```

Comment:
That's a totally correct implementation! You can get rid of `()` here, but that's very minor. It already looks great 👍🏻 
```suggestion
andM fa fb = fa >>= \a -> if a then fb else pure False
```

Reply:
Thank you. I changed as your suggestion.